jekyll/_cci2/runner-on-kubernetes.adoc

```diff
@@ -0,0 +1,94 @@
+---
+version:
+- Cloud
+- Server v3.x
+---
+= CircleCI Runner on Kubernetes
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+NOTE: CircleCI runner is available on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative (or https://circleci.com/contact-us/?cloud[contact us]) for information on how to sign up for the Scale Plan.
+
+toc::[]
+
+== Introduction
+
+This installation guide is to help setup CircleCI's runner(s) on your Kubernetes cluster. We provide a Helm chart which makes installing the pods easy.
+
+The Helm chart will spin up one or more pods of *the same runner resource class*. This is useful for when you want all of these runners to execute jobs requesting the same execution environment. Each runner will pull jobs off the queue on an as-available basis.
+
+If you want to have different runners specialized for different workloads, it is recommended to create different runner resource classes and rerun these instructions/have separate charts for each runner class you create.
+
+=== Prerequisites
+* Be on Scale plan or trialing Scale plan. See above.
+* Have a Kubernetes cluster up and running where you'd like to deploy your runner(s).
+* Go through the steps to <<runner-installation#authentication,create and authenticate a CircleCI runner resource class>>.
+
+=== Pending Work
+* The Helm chart currently doesn't configure permissions, so the containers will not have elevated privileges. If you need elevated permissions (e.g., for Docker in Docker work, etc.), you will need to modify the chart yourself.
+* The Helm chart currently doesn't support auto-scaling - you will need to modify the `+replicaCount+` parameter inside `+values.yaml+` yourself and re-apply the chart to your cluster.
+* The Helm chart currently expects only one runner resource class type and one token. If you want to configure clusters for multiple runner resource classes, you'll need to set up separate charts for each one.
+
+== Getting Started
+
+. Clone the repository at https://github.com/circleci/circleci-runner-k8s.
```

Comment:
Will this location be in CircleCI-Public now?

Reply:
Good point, pushing fix for this now.